From khali@linux-fr.org Sun Sep  3 13:23:01 2006
Date: Sun, 3 Sep 2006 22:22:50 +0200
From: Jean Delvare <khali@linux-fr.org>
To: Greg KH <greg@kroah.com>
Cc: Linux I2C <i2c@lm-sensors.org>, Adrian Bunk <bunk@stusta.de>
Subject: [PATCH 05/13] i2c-algo-pcf: Discard the mdelay data struct member
Message-Id: <20060903222250.17527c20.khali@linux-fr.org>
Content-Disposition: inline; filename=i2c-algo-pcf-kill-mdelay.patch

From: Adrian Bunk <bunk@stusta.de>

i2c-algo-pcf: Discard the mdelay data struct member

Just as i2c-algo-bit, i2c-algo-pcf has an unused mdelay struct member,
which we can get rid of to spare some code and memory.

Signed-off-by: Adrian Bunk <bunk@stusta.de>
Signed-off-by: Jean Delvare <khali@linux-fr.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/i2c/busses/i2c-elektor.c |    1 -
 include/linux/i2c-algo-pcf.h     |    1 -
 2 files changed, 2 deletions(-)

--- gregkh-2.6.orig/drivers/i2c/busses/i2c-elektor.c
+++ gregkh-2.6/drivers/i2c/busses/i2c-elektor.c
@@ -196,7 +196,6 @@ static struct i2c_algo_pcf_data pcf_isa_
 	.getclock   = pcf_isa_getclock,
 	.waitforpin = pcf_isa_waitforpin,
 	.udelay	    = 10,
-	.mdelay	    = 10,
 	.timeout    = 100,
 };
 
--- gregkh-2.6.orig/include/linux/i2c-algo-pcf.h
+++ gregkh-2.6/include/linux/i2c-algo-pcf.h
@@ -35,7 +35,6 @@ struct i2c_algo_pcf_data {
 
 	/* local settings */
 	int udelay;
-	int mdelay;
 	int timeout;
 };
 
